lms/templates/instructor/instructor_dashboard_2/cohorts.underscore

```diff
@@ -34,17 +34,17 @@
         <hr class="divider divider-lv1" />
 
         <!-- Uploading a CSV file of cohort assignments. -->
-        <a class="toggle-cohort-management-secondary" href="#cohort-management-file-upload"><%- gettext('Assign students to cohorts by uploading a CSV file') %></a>
-        <div class="cohort-management-file-upload csv-upload is-hidden" id="cohort-management-file-upload"></div>
+        <button class="toggle-cohort-management-secondary" data-href="#cohort-management-file-upload"><%- gettext('Assign students to cohorts by uploading a CSV file') %></button>
+        <div class="cohort-management-file-upload csv-upload is-hidden" id="cohort-management-file-upload" tabindex="-1"></div>
 
         <div class="cohort-management-supplemental">
             <p class="">
                 <span class="icon fa fa-info-circle" aria-hidden="true"></span>
                 <%= HtmlUtils.interpolateHtml(
                     gettext('To review student cohort assignments or see the results of uploading a CSV file, download course profile information or cohort results on {link_start} the Data Download page. {link_end}'),
```

Comment:
I know this was pre-existing, but can you move "the" outside of the button text? Possibly also "page". It looks a bit odd now that it is a button.

![image](https://cloud.githubusercontent.com/assets/484484/17786323/47131aaa-6552-11e6-83ff-0b794f7c0268.png)